documentation/modules/ROOT/pages/connectors/vitess.adoc

```diff
@@ -1771,6 +1771,10 @@ See xref:vitess-data-types[how Vitess connectors map data types] for the list of
 |`io.debezium.schema.SchemaTopicNamingStrategy`
 |The name of the TopicNamingStrategy class that should be used to determine the topic name for data change, schema change, transaction, heartbeat event etc., defaults to `SchemaTopicNamingStrategy`.
 
+|[[vitess-property-override-data-change-topic-prefix]]<<vitess-property-override-data-change-topic-prefix, `override.data.change.topic.prefix`>>
+|No default
+|If the topic naming strategy `io.debezium.connector.vitess.TableTopicNamingStrategy` is used, this property is used in place of `topic.prefix` to create the data change topic name.
```

Comment:
```suggestion
|Specifies the prefix that the connector uses to create the names of data change topics when xref:vitess-property-topic-naming-strategy[topic.naming.strategy] is set to `io.debezium.connector.vitess.TableTopicNamingStrategy`.
The connector uses the value of this property instead of the specified xref:vitess-property-topic-prefix[`topic.prefix`]. 

```